### PR TITLE
Add external link for Environment Variables

### PR DESCRIPTION
**Pull request by kmmelcher**
This link will be useful if the user doesn't know how to set Environment Variables

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -92,8 +92,7 @@ How?
 
 1. Get a personal access token with permission to manage your *gists*
    from `GitHub settings <https://github.com/settings/tokens>`__
-2. Set an environment variable called ``GETGIST_TOKEN`` with your
-   personal access token
+2. Set an `environment variable <https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-environment-variables-in-linux/>`__ called                  ``GETGIST_TOKEN`` with your personal access token
```
Comment (cuducos) [inline README.rst:95]: Many thanks, @kmmelcher : )

Here is one os my preferred conundrums of documenting open-source packages: whether we should everything or limit the documentation to what the package actually does.

In this specific case, creating environmental variables is not something directly related to this package, although it is still important for a nicer UX in this package. Then, thinking about that, we decide to document it. However, now we have to think about Windows, macOS, and GNU/Linux users. And to consider they might be using Bash, Zsh, or PowerShell… and then, to cover that all, we ended up writing more and more documentation that is not related to this package — our task ends up being teaching users how to use their OS and tooling and not to teach them how to use GetGist. And we are back to step 1.

Long story short, we just need to be clear about where are the boundaries. What do you think about slightly changing the wording and formatting? I suggest keeping this line as it is and adding another one like:

> This [article](https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-environment-variables-in-linux) might help you create an environment variable in a Unix-based operational system with Bash, but feel free to search alternatives for other systems and shells.

File: README.rst [hunk 2]
```diff
@@ -92,8 +92,10 @@ How?
 
 1. Get a personal access token with permission to manage your *gists*
    from `GitHub settings <https://github.com/settings/tokens>`__
-2. Set an environment variable called ``GETGIST_TOKEN`` with your
-   personal access token
+2. Set an environment variable called ``GETGIST_TOKEN`` with your personal access token
+
+
+>This `article <https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-environment-variables-in-linux/>`__ might help you create an environment >variable in a Unix-based operational system with Bash, but feel free to search alternatives for other systems and shells.
```
Comment (cuducos) [inline README.rst:98]: Hi @kmmelcher. Viewing the [redered version of the `README.rst`](https://github.com/cuducos/getgist/blob/f04504b8625a2b48f09d589e4c79a5ca954dbb80/README.rst) it looks like there are two `>` characters seem like they shouldn't be there. Also, there's an unneeded extra blank line.

What do yo think about this minor edits?

```suggestion

This `article <https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-environment-variables-in-linux/>`__ might help you create an environment variable in a Unix-based operational system with Bash, but feel free to search alternatives for other systems and shells.
```

**Comment by kmmelcher** [comment]
All Right, sorry for mistyping, that way seems good!

**Comment by cuducos** [comment]
Ainda ficou um erro de digitação (`>variable` ao invés de `variable`), mas fiz o merge e corrigi depois ; )
